### PR TITLE
fix: threads profile picture url if not available and update readme

### PR DESCRIPTION
**Pull request by Akmalfauzi**

### CODE REVIEW

**Comment by atymic** [comment]
I don't see any changes here. Arr::get already returns null?

**Comment by Akmalfauzi** [comment]
Yes, that's right, because in my case, it only returns the id and username fields.

So, I fixed the undefined index "threads_profile_picture_url" error so that it could still run even though the "threads_profile_picture_url" field did not exist or was empty.

**Comment by Akmalfauzi** [comment]
I also fixed the typo in the `README.md` file when adding code to the `EventServiceProvider.php` file

Screenshot 2024-08-21 at 08 06 42

**Comment by atymic** [comment]
`Arr::get` will return null already if the field does not exist?

**Comment by Akmalfauzi** [comment]
Yes, that's right.

File: src/Threads/Provider.php [hunk 1]
```diff
@@ -82,7 +82,7 @@ protected function mapUserToObject(array $user)
             'nickname'     => Arr::get($user, 'username'),
             'name'         => null,
             'email'        => null,
-            'avatar'       => Arr::get($user, 'threads_profile_picture_url'),
+            'avatar'       => Arr::get($user, 'threads_profile_picture_url') ?? null,
```
Comment (atymic) [inline src/Threads/Provider.php:85]:
```suggestion
            'avatar'       => Arr::get($user, 'threads_profile_picture_url'),
```

No need to add the null coalesce

**Review by atymic — CHANGES_REQUESTED** [review]
Just this change then it's good merge.

File: src/Threads/Provider.php (hunk 1 above)
Comment (Akmalfauzi) [reply src/Threads/Provider.php:85]: hmm, oke thank you. i will fix soon.

**Review by atymic — APPROVED** [review]

**Pull request merged**